resources/assets/js/features/search/layouts/SearchIndex.js

```diff
@@ -17,9 +24,32 @@ export default localize(class SearchIndex extends React.Component {
         const {translate} = this.props;
         return (
             <div>
-                <PageHeader header={translate('search_index.find_a_ride_header')}/>
-                <SearchForm pageType="index" />
-                <WizardTrip />
+                <section className="home-slider">
+                    <div className="home-slider__tabs wizard-tabs">
+                        <a href="#" onClick={() => {this.setState({mode: 'passenger'})}}
+                           className={'wizard-tabs__tab ' + (this.state.mode === 'passenger' ? 'wizard-tabs__tab_active' : '')}
+                        >
+                            <img src="/template/img/welcome-icon1.png" alt=""/>
+                                <span>{translate('search_index.i_am_passenger')}</span>
+                        </a>
+                        <a href="#" onClick={() => {this.setState({mode: 'driver'})}}
+                           className={'wizard-tabs__tab ' + (this.state.mode !== 'passenger' ? 'wizard-tabs__tab_active' : '')}
```

Comment:
avoid using such statements in templates, create method that will return bool value based on local state